src/modules/Builder/features/ActivitySettings/SubscalesConfiguration/SubscalesConfiguration.const.ts

```diff
@@ -77,11 +78,27 @@ export const genderItem = getNewActivityItem({
   },
 });
 
-export const ageItem = getNewActivityItem({
+export const ageTextItem = getNewActivityItem({
   allowEdit: false,
   name: LookupTableItems.Age_screen,
   question: t('ageQuestion'),
-  config: defaultTextConfig,
+  config: {
+    ...defaultTextConfig,
+    maxResponseLength: 3,
+    numericalResponseRequired: true,
+  },
   responseType: ItemResponseType.Text,
   responseValues: null,
 });
+
+export const ageDropdownItem = getNewActivityItem({
+  allowEdit: false,
+  name: LookupTableItems.Age_screen,
+  question: t('ageQuestion'),
+  config: defaultNumberSelectionConfig,
+  responseType: ItemResponseType.NumberSelection,
+  responseValues: {
+    minValue: 1,
```

Comment:
The AC states a range of 0-50 🤷🏻‍♂️ 
```suggestion
    minValue: 0,
```

Reply:
Oh you're right, but that feels a bit weird. Let me clarify that real quick

Reply:
Also FWIW, `0` is also an acceptable value when age is a text response type.

Reply:
Confirmed that it should be 0. Thanks for catching this Paul 🙏

Reply:
> Also FWIW, 0 is also an acceptable value when age is a text response type.

Ah yeah, I noticed that after you pointed this out so I would've had to address that if it was supposed to be 1. But I guess that's not necessary